blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -2,11 +2,27 @@
 
 import React, { Component, PropTypes } from 'react';
 import { Icon } from '@jenkins-cd/react-material-icons';
+import { logging } from '@jenkins-cd/blueocean-core-js';
 import { ExpandablePath, ReadableDate, LiveStatusIndicator, TimeDuration } from '@jenkins-cd/design-language';
 import ChangeSetToAuthors from './ChangeSetToAuthors';
-import moment from 'moment';
+import { TimeManager } from '../util/serverBrowserTimeHarmonize';
+
+const logger = logging.logger('io.jenkins.blueocean.dashboard');
```

Comment:
`io.jenkins.blueocean.dashboard.RunDetailsHeader` you mean? Does that mean I have to set the debug level for that specific component or do I get  everything "under" `'io.jenkins.blueocean.dashboard'`